packages/user-extension/package.json

```
@@ -0,0 +1,61 @@
+{
+  "name": "@jupyterlab/user-extension",
+  "version": "3.3.0-alpha.15",
```

Let's update to `4.0.0-alpha.1` as a follow-up to https://github.com/jupyterlab/jupyterlab/pull/11537:

```suggestion
  "version": "4.0.0-alpha.1",
```